platform/android/app/src/test/kotlin/coop/polypoly/polypod/features/FeatureStorageTest.kt

```diff
@@ -40,6 +40,15 @@ private fun createMockFeaturePackage(parent: File, child: String): File {
     return featurePackage
 }
 
+@Ignore(
```

Comment:
Ignoring means that you might want to un-ignore it later, right? In that case, maybe create a ticket with the pre-conditions to set it back to work again?

Reply:
Sort of, it may be gone completely after the whole FeatureStorage stuff is moved to core, it can be part of that ticket.